### PR TITLE
LPD-17794 [BLOCKER] Cannot cancel the creation of a KB Article

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Issue https://liferay.atlassian.net/browse/LPD-17794 

**Steps to reproduce:** 
1. Go to Knowledge Base
2. Create an article
3. Click on Cancel or back button

**Solution proposed:** 
It was trying to unlock the article in the cancel action, but when the article is new there is no unlock, so we can just redirect to the previous page.

### CODE REVIEW

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/EditKBArticleDisplayContext.java [hunk 1]
```diff
@@ -107,6 +107,12 @@ public Map<String, String> getAvailableKBArticleSections()
 	}
 
 	public String getCancelURL() {
+		long resourcePrimKey = getResourcePrimKey();
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/EditKBArticleDisplayContext.java:110]: this should be 

if FF > return getRedirect

after that, maybe to check the resourcePrimKey, we should check with or without the FF

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:reevaluate:1287401_9050

**Comment by ambrinchaudhary** [comment]
ci:report:2538927400

**Comment by ambrinchaudhary** [comment]
No unique failures for this pull. I'm forcing the forward...

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Pull request closed without merging**